docs/_data/2017.na.speakers.yaml

```diff
@@ -1090,6 +1090,7 @@
     title: Intelligent Documents and the Verifiability Crisis in Science, Tech Writing,
         and Life
     year: 2017
+    youtubeId: kOzQPpc-KDo
```

Comment:
wonder if you need to quote the id because of the hyphen?

Reply:
No, that's working fine